dockers/actions-plan-preview/main.go

```diff
@@ -77,7 +77,7 @@ func main() {
 		return
 	}
 
-	if !event.PRMergeable {
+	if event.PRMergeable != nil && *event.PRMergaeable == false {
```

Comment:
```suggestion
	if event.PRMergeable != nil && !*event.PRMergaeable {
```

Reply:
👍

Reply:
Compared to `!*`, I think the current one is clearer. Only me. 😄

Reply:
Well, what I can say is I would like to omit `== bool` since it is redundant to me, but get your point of the current method is more clear than that. So, it's up to you 🙆‍♀️

Reply:
It is. I would respect your feeling.